test/UnbrandedProjects/NoTest-TypeSpec/src/Generated/Models/Thing.cs

```diff
@@ -73,6 +73,20 @@ internal Thing(string name, BinaryData requiredUnion, ThingRequiredLiteralString
         /// To assign an already formatted json string to this property use <see cref="BinaryData.FromString(string)"/>.
         /// </para>
         /// <para>
+        /// <remarks>
```

Comment:
Can you add a few screenshots of the intellisense in vscode and vs in the description?